src/components/alertMessage/AlertMessage.styles.ts

```diff
@@ -78,19 +78,25 @@ export const Message = styled.div<{
   letter-spacing: 0.019px;
 `;
 
-export const AlertLink = styled(NavLink)<{
+export const AlertLink = styled.p<{
```

Comment:
can we use span here ?
```suggestion
export const AlertLink = styled.span<{
```